### PR TITLE
Minor code fixes

### PR DESCRIPTION
**Pull request by cijothomas**
Minor improvements.

### CODE REVIEW

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ActivityExtensions.cs [hunk 1]
```diff
@@ -143,7 +143,8 @@ internal static OtlpTrace.Span ToOtlpSpan(this Activity activity)
             {
                 Name = activity.DisplayName,
 
-                Kind = (OtlpTrace.Span.Types.SpanKind)(activity.Kind + 1), // TODO: there is an offset of 1 on the enum.
+                // There is an offset of 1 on the Otlp enum.
```
Comment (cijothomas) [inline src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ActivityExtensions.cs:146]: Don't see any more TODOs here..

**Review by eddynaka — APPROVED** [review]

File: src/OpenTelemetry/Logs/OpenTelemetryLogger.cs [hunk 2]
```diff
@@ -52,6 +52,7 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
             this.provider.Processor?.OnEnd(record);
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```
Comment (cijothomas) [inline src/OpenTelemetry/Logs/OpenTelemetryLogger.cs:55]: Also thinking if we should remove this check completely! If we remove it, then `logger.Log(LogLevel.None, "I dont want to see this")` will appear in logs. :-/

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ActivityExtensions.cs (hunk 1 above)
Comment (reyang) [inline src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ActivityExtensions.cs:146]:
```suggestion
                // There is an offset of 1 on the OTLP enum.
```

**Review by reyang — APPROVED** [review]
LGTM with one minor suggestion.

**Pull request merged**